components/icons/Icon.tsx

```diff
@@ -21,12 +21,14 @@ import * as React from 'react';
 import { Theme, ThemeConsumer } from '../theme';
 
 export interface IconProps {
+  'aria-label'?: string;
```

Comment:
As we're now basically allowing _any_ prop, I wonder if we should restrict to `aria-label` 🤔 ? Sometimes, `aria-hidden` could be interesting as well, for instance. Should we do something like:

```
export interface IconProps extends React.AriaAttributes {
```

Or even:

```
export interface IconProps extends React.HTMLAttributes<{}> {
```

?

Reply:
Ah yeah good idea, I'll extends the `React.AriaAttributes`!